content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -202,21 +258,31 @@ Teams:
 
 ## 7. Configure continuous deployment to Netlify
 
-From the repository's folder, run `netlify init` to configure a new Netlify site:
+You can now create your Netlify site and configure continuous deployments for it.
+
+### Steps
+
+1. From the repository's folder, run `netlify init`.
+
+   ```terminal
+   netlify init
+   ```
+
+2. Use the configuration prompts to configure your Netlify site.
 
-![Running netlify init to configure a new Netlify site](https://res.cloudinary.com/prismaio/image/upload/v1628765126/docs/Ctt56FN_oyu0dq.png)
+   ![Running netlify init to configure a new Netlify site](images/500-05-deploying-to-netlify-netlify-init-configure-site.png)
 
-You will be prompted with configuration options:
+   1. Select `Create & configure a new site`.
+   2. If prompted to select your team, pick the one in which Netlify will create the site.
+   3. Give the site a unique name.
+   4. Authorize Netlify with your GitHub account.<br />
+      Netlify uses your GitHub account to configure notification webhooks and deploy keys and uses these to configure the continuous integration for the app.
+   5. _Your build command_: `npm run build`
+   6. _Directory to deploy_: `public`
 
-1. Select `Create & configure a new site`.
-2. If prompted to select your team, pick the one in which Netlify will create the site.
-3. Give the site a unique name.
-4. Authorize Netlify with your GitHub account.<br />
-   Netlify uses your GitHub account to configure notification webhooks and deploy keys and uses these to configure the continuous integration for the app.
-5. _Your build command_: `npm run build`
-6. _Directory to deploy_: `public`
+### Result
 
-**Checkpoint:** The Netlify CLI should output the following:
+The Netlify CLI should show the following:
```

Comment:
```suggestion
Check that the Netlify CLI shows the following:
```

[Our style guide says to avoid the word "should".](https://www.prisma.io/docs/about/prisma-docs/style-guide/word-choice#avoid-ambiguous-english-words)